### ISSUE TITLE
Next release.. when?

### ISSUE THREAD

**Issue by kernel72** [jquense/react-bootstrap-modal#53]
Hi! Are there any plans or timings to release next version? As I see, fixes of React's warnings (like propTypes) were already committed and it will be great to see them officially. Can you orient when next version will be published? Thank you!

**Comment by webyonet** [comment]
+1

**Comment by SkoebaSteve** [comment]
Yes please, we're currently using the git repo as a source but I would like to switch back to using the npm package properly. The build works, so if master could be published to npm that would be great!
